tests/SignalsHandlerTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace React\Tests\EventLoop;
+
+use React\EventLoop\SignalsHandler;
+
+final class SignalsHandlerTest extends TestCase
+{
+    public function testEmittedEventsAndCallHandling()
```

Comment:
Signal constants are only defined when PCNTL is loaded: http://php.net/manual/en/pcntl.constants.php

This test should probably check for this first. Also, maybe add a remark about this to the README?

Reply:
Defining in `tests/bootstrap.php` when they don't exist in case `PCNTL` isn't loaded